examples/demo/public/environment.dev.json

```diff
@@ -1,3 +1,4 @@
 {
-  "apiBaseUrl": "http://localhost:5000/api/{path}"
+  "apiBaseUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}",
```

Comment:
yep oussama,
 in dev mode:
"apiBaseUrl": "http://localhost:5000/api/{path}"

is the good one ^^

Very nice PR !